yb-voyager/cmd/importSchema.go

```diff
@@ -81,7 +82,18 @@ func importSchema() {
 	}
 	objectList = applySchemaObjectFilterFlags(objectList)
 	log.Infof("List of schema objects to import: %v", objectList)
-	importSchemaInternal(&target, exportDir, objectList, nil)
+
+	skipFn := func(objType, stmt string) bool {
+		stmt = strings.ToUpper(stmt)
+		return strings.HasPrefix(stmt, "ALTER TABLE")
+	}
+	importSchemaInternal(&target, exportDir, objectList, skipFn)
+
+	skipFn = func(objType, stmt string) bool {
+		stmt = strings.ToUpper(stmt)
+		return !strings.HasPrefix(stmt, "ALTER TABLE")
+	}
```

Comment:
You can frame the implementation as a complement of the previous function e.g.

    skipFn = func(objType, stmt string) bool {
        return !isAlterTableStmt(objType, stmt)
    }

This implementation declares the "intent" of the code. i.e. we're importing the statements skipped by the previous invocation.